install_config/aggregate_logging.adoc

```diff
@@ -1653,34 +1653,50 @@ Use:
 
 ----
 config.yaml: |
-  myapp-dev:
+
+# uncomment and use this to override the defaults from env vars
+#.defaults: <1>
+#  delete:
+#    days: 30
+#  runhour: 0
+#  runminute: 0
+
+  myapp-dev: <2>
     delete:
       days: 1
 
-  myapp-qe:
+  myapp-qe: <3>
     delete:
       weeks: 1
 
-  .operations:
+  .operations: <4>
     delete:
       weeks: 8
 
-  .defaults:
+  .defaults: <5>
     delete:
       days: 31
     runhour: 0
     runminute: 0
     timezone: America/New_York
 
   .regex:
-    - pattern: '^project\..+\-dev\..*$'
+    - pattern: '^project\..+\-dev\..*$' <6>
       delete:
-        days: 15
-    - pattern: '^project\..+\-test\..*$'
+        days: 1
+    - pattern: '^project\..+\-test\..*$' <7>
       delete:
         days: 30
 ----
 
+<1> Optionally, change the default number of days between run and the run hour and run minute. 
+<2> Delete indices in the *myapp-dev* project older than `1 day`
+<3> Delete indices in the *myapp-qe* project older than `1 week`
+<4> Delete *operations* logs older than `8 weeks`
+<5> Delete all other projects indices after they are `31 days` old
+<6> Delete indices older than 1 day that are matched by the '^project\..+\-dev.*$' regex
+<7> Delete indices older than 2 days that are matched by the '^project\..+\-test.*$' regex
```

Comment:
It is days: 30 above.

Reply:
@anpingli, for item 5? The `days` parameter is `31`, so I'm confused. Thanks.

```
  .defaults: <5>
    delete:
      days: 31
    runhour: 0
    runminute: 0
    timezone: America/New_York
```